src/yunohost/data_migrations/0006_migrate_pwd.py

```diff
@@ -0,0 +1,60 @@
+import spwd
+import crypt
+import random
+import string
+import subprocess
+
+from moulinette import m18n
+from moulinette.core import MoulinetteError
+from moulinette.utils.log import getActionLogger
+from moulinette.utils.process import run_commands, check_output
+from moulinette.utils.filesystem import append_to_file
+from moulinette.authenticators.ldap import Authenticator
+from yunohost.tools import Migration
+
+logger = getActionLogger('yunohost.migration')
+SMALL_PWD_LIST = ["yunohost", "olinux"]
+
+class MyMigration(Migration):
+    "Migrate password"
+
+    def migrate(self):
+
+        if self._is_root_pwd_listed(SMALL_PWD_LIST):
+            new_hash = self._get_admin_hash()
+            self._replace_root_hash(new_hash)
```

Comment:
Rethinking about this, I think we need to clarify the purpose of this migration ? (Or maybe I'm kinda lost about it)

It seems like this PR does two different things : 
- check if the root password is "bad", and if so, resync it with the admin password
- change tools_adminpw such that it also changes the root password each time the admin password is modified...

But for existing instance on which the root password ain't "bad", after this migration, we still end up with the root password and admin password being desynchronized. Shouldn't we sync the root and admin password in any case so that we end up with consistent setups in all cases ?